core/src/main/resources/swagger/rawls.yaml

```diff
@@ -207,6 +207,92 @@ paths:
             - email
             - profile
 
+  '/api/billing/{projectId}/googleRole/{role}/{email}':
+    put:
+      responses:
+        '200':
+          description: Successfully Granted Google Role to User
+        '403':
+          description: You must be a project owner granting a role on the whitelist
+          schema:
+            $ref: '#/definitions/ErrorReport'
+        '404':
+          description: User not found
+          schema:
+            $ref: '#/definitions/ErrorReport'
+        '500':
+          description: Rawls Internal Error
+          schema:
+            $ref: '#/definitions/ErrorReport'
+      description: grant a google role for a user in the billing project the caller owns
```

Comment:
You probably want to mention that their pet gets access too.

Reply:
done